From: Manuel Quintero Fonseca <sakunix@yahoo.com>
Subject: resource: downgrade "resource sanity check" warning to debug level
Date: Fri, 22 May 2026 17:01:31 -0700

The "resource sanity check" warning currently triggers on certain systems,
causing unnecessary noise in the kernel logs for users.  This patch
downgrades the log level from pr_warn to pr_debug.

This change reduces log clutter while keeping the diagnostic information
available for debugging purposes if needed.

Link: https://lore.kernel.org/20260523000131.7086-1-sakunix@yahoo.com
Signed-off-by: Manuel Quintero Fonseca <sakunix@yahoo.com>
Cc: Andriy Shevchenko <andriy.shevchenko@linux.intel.com>
Cc: Bjorn Helgaas <bhelgaas@google.com>
Cc: Hans de Goede <hansg@kernel.org>
Cc: Mika Westeberg <mika.westerberg@linux.intel.com>
Cc: "Rafael J. Wysocki" <rafael@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 kernel/resource.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/kernel/resource.c~resource-downgrade-resource-sanity-check-warning-to-debug-level
+++ a/kernel/resource.c
@@ -1859,7 +1859,7 @@ int iomem_map_sanity_check(resource_size
 		if (p->flags & IORESOURCE_BUSY)
 			continue;
 
-		pr_warn("resource sanity check: requesting [mem %pa-%pa], which spans more than %s %pR\n",
+		pr_debug("resource sanity check: requesting [mem %pa-%pa], which spans more than %s %pR\n",
 			&addr, &end, p->name, p);
 		err = -1;
 		break;
_
